compilation_tests/ctest_standard_layout.cpp

```diff
@@ -167,48 +167,35 @@ MDSPAN_STATIC_TEST(
 
 MDSPAN_STATIC_TEST(
   std::is_standard_layout<
-    stdex::layout_stride<stdex::dynamic_extent, 1, 3>::template mapping<
+    stdex::layout_stride::template mapping<
       stdex::extents<42, stdex::dynamic_extent, 73>
     >
   >::value
 );
 
 MDSPAN_STATIC_TEST(
   std::is_standard_layout<
-    stdex::layout_stride<stdex::dynamic_extent, 1, 3>::template mapping<
+    stdex::layout_stride::template mapping<
       stdex::extents<42, 27, 73>
     >
   >::value
 );
 
 MDSPAN_STATIC_TEST(
   std::is_standard_layout<
-    stdex::layout_stride<12, 1, 3>::template mapping<
-      stdex::extents<42, stdex::dynamic_extent, 73>
-    >
-  >::value
-);
-
-MDSPAN_STATIC_TEST(
-  std::is_standard_layout<
-    stdex::layout_stride<1, 2, 3>::template mapping<
-      stdex::extents<1, 2, 3>
-    >
-  >::value
-);
-
-MDSPAN_STATIC_TEST(
-  std::is_standard_layout<
-    stdex::layout_stride<stdex::dynamic_extent, stdex::dynamic_extent>::template mapping<
+    stdex::layout_stride::template mapping<
       stdex::extents<stdex::dynamic_extent, stdex::dynamic_extent>
     >
   >::value
 );
 
+// TODO: Remove this test alltogether?
```

Comment:
I think this could still be standard_layout? But maybe not. In either case it doesn't pass on GCC anymore.